README.md

```diff
@@ -87,6 +87,27 @@ For instance, if we only wanted to retrieve **2 images** here is what the call w
 All the calls are **Asynchronous**, which means they will return a **Promise** object, making it a bit more flexible in order to adjust to any kind of application.
 Again, for a more thorough example there is a sample [application use case](sample/sample.php) in this repo.
 
+To run the [sample upload application code](https://github.com/Bynder/bynder-php-sdk/blob/master/sample/sampleUploadApplication.php) add ```sample_config.ini``` to the root directory of the project.
+You may choose either ```oauth2``` or ```permanentTokens``` as the authentication method.
```

Comment:
just as a note,`permanentTokens` is not supported for the new FS

Reply:
Yeah Huib mentioned this 😄  I have made some changes to address this and currently am testing them out before a push.